api/controllers/user-environment-variable.js

```diff
@@ -0,0 +1,79 @@
+const { userEnvVar } = require('../../config');
+const { wrapHandlers } = require('../utils');
+const { serialize, serializeMany } = require('../serializers/user-environment-variable');
+const { encrypt } = require('../services/Encryptor');
+const { ValidationError } = require('../utils/validators');
+const { Site, UserEnvironmentVariable } = require('../models');
+
+function validate({ name, value }) {
```

Comment:
potentially refactor this and some other logic to a dedicated UserEnvironmentVariableService.